index.html

```diff
@@ -273,132 +273,141 @@ <h2>About Me</h2>
+                    <img
+                      src="./assets/icons/Rectangle 56.svg"
+                      alt="language-icon"
+                    />
+                    <h3>Frameworks</h3>
+                  </div>
+                  <div class="stack-skills">
+                    <ul>
+                      <li>Bootstrap</li>
+                      <li>Ruby On Rails</li>
+                      <li>RsPec</li>
+                      <li>CapyBara</li>
+                      <li>Selenium</li>
+                    </ul>
+                  </div>
+                </li>
+                <li>
+                  <div class="stack-heading stack-skills_styled">
+                    <img
+                      src="./assets/icons/Rectangle 57.svg"
+                      alt="language-icon"
+                    />
+                    <h3>skills</h3>
+                  </div>
+                  <div class="stack-skills">
+                    <ul>
+                      <li>Codekit</li>
+                      <li>Github</li>
+                      <li>Codepen</li>
+                      <li>Gitlab</li>
+                      <li>Terminal</li>
+                    </ul>
+                  </div>
+                </li>
+              </ul>
             </div>
           </div>
         </section>
 
         <section class="contact-info">
           <h3>
-            I'm always interested in hearing about new projects, so if you'd
-            like to chat please get in touch.
+           If you have an application you are interested in developing, a feature that you need built or a project that needs coding.
           </h3>
-          <form action="mailto:ekiugbotega@gmail.com" method=”POST” enctype=”text/plain” class="contact-form">
+          <form
+            action="https://formspree.io/f/mqkoqlor"
+            method="”POST”"
+            class="contact-form"
+          >
             <input
               type="text"
               name="name"
               placeholder="Full name"
               class="contact form-info"
+              required
+              maxlength="30"
             />
             <input
               type="email"
               name=""
               id=""
               placeholder="Email address"
-              class="contact form-info "
+              class="contact form-info"
+              required
+              minlength="30"
             />
             <textarea
-              name=""
-              id=""
+              name="text"
+              id="text"
               cols="10"
-              rows="4"
+              rows="5"
               placeholder="Enter text here"
               class="contact text"
+              maxlength="500"
+              required
             ></textarea>
             <div class="submit">
-            <input type="submit" value="Get In Touch" class="btn-submit">
+              <input type="submit" value="Get In Touch" class="btn-submit" />
             </div>
```

Comment:
- [ ] Kindly ensure your that your button follows the button interaction  when it is `hovered`, `active`,... I also suggest you apply the same styling on other buttons such as `Get My Resume`, `See Project` buttons,...
You can all the infrormation you nee from this image below on your chosen Figma tremplate
![Screenshot from 2023-02-21 17-37-11](https://user-images.githubusercontent.com/83042742/220390392-eab116bf-174c-4b3e-b99a-2fdbfa40c389.png)